### ISSUE TITLE
php artisan config:cache generate error

### ISSUE THREAD

**Issue by dels07** [fisharebest/laravel-assets#13]
Hi,

I tried to use **php artisan config:cache** in production environment, however this package throw up an error:
``` php
[Symfony\Component\Debug\Exception\FatalThrowableError]
  Call to undefined method Fisharebest\LaravelAssets\Filters\RewriteCssUrls::__set_state()
```
Likely due to the use of Closure in **config/assets.php** file, here the generated **bootstrap/cache/config.app**:
``` php
.....
'assets' => 
  array (
    'enabled' => true,
    'css_source' => 'css',
    'js_source' => 'js',
    'destination' => 'min',
    'destination_url' => '',
    'css_filters' => 
    array (
      0 => 
      Fisharebest\LaravelAssets\Filters\RewriteCssUrls::__set_state(array(
      )),
      1 => 
      Fisharebest\LaravelAssets\Filters\MinifyCss::__set_state(array(
      )),
    ),
    'js_filters' => 
    array (
      0 => 
      Fisharebest\LaravelAssets\Filters\FinalNewline::__set_state(array(
      )),
      1 => 
      Fisharebest\LaravelAssets\Filters\MinifyJs::__set_state(array(
      )),
    ),
    'loader' => 
    Fisharebest\LaravelAssets\Loaders\FileGetContents::__set_state(array(
    )),
    'notifiers' => 
    array (
    ),
    'gzip_static' => 6,
    'inline_threshold' => 0,
.....
```
Currently I disabled config:cache in production, is there any way to fix this?

**Comment by fisharebest** [comment]
Should be easy to fix.

**Comment by fisharebest** [comment]
Fixed in 1.6.0
